### DISCUSSION TITLE
idxchannel.com - Apakah AdBlockID Dapat Melakukan Redirect Ke Halaman "View All"?

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1142] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/19517680/156814936-a7e12500-9702-4266-84cc-278b8a870273.png)

https://www.idxchannel.com/ecotainment/intip-10-sumber-kekayaan-maharani-kemala-crazy-rich-bali-pemilik-ms-glow/all

bisa tidak adblockid meredirect setiap page website berita menjadi "view all"

dari sudut pembaca saya paling kesal haris lihat page 1,2,3 ... 7, 
lebih nyaman tinggal scroll ke bawah tanpa harus click sampai  7x, yg hal ini sangat tidak efisien, diantaranya :
- mengkonsumsi banyak energi
- mengkonsumsi bandwith (kuota) internet
- menunggu loading setiap load page
- bikin capek jari

**Comment by realodix** [comment, marked as answer]
Tidak bisa Gan

Dalam kasus ini, ini butuh plugin khusus yang bisa redirect dari `example.com/contoh-artikel` ke `example.com/contoh-artikel/all`.
